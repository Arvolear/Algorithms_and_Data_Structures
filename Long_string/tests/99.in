5880102508731945220587449246587
953085785417374796593