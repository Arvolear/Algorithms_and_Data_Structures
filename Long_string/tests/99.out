5880102509685031006004824043180
5880102507778859435170074449994
5604242117869461953668571869134934611057960224478091
6169541712